app/pages/EditBreakingNewsPage.scss

```diff
@@ -0,0 +1,58 @@
+.edit-breaking-news-header, .edit-breaking-news-footer {
+    display: flex;
+    justify-content: center;
+}
+
+.edit-breaking-news-header {
+    font-size: 28px;
+    font-weight: 600;
+    padding: 10px 0px 0px;
+}
+
+.edit-breaking-news-footer {
+    padding-bottom: 20px;
```

Comment:
Use margin, not padding, to separate elements from each other. The difference would be much more obvious if you had a background color or a border on the element, but there are other subtle reasons why you should still prefer margin for separate elements even if you don't have a background color or a border.